### PR TITLE
Survey data and Bipolar scheme

### LINKED ISSUE #47: plotERP should elegantly handle when electrode meta-data is missing

### PR DESCRIPTION
**Pull request by luiztauffer**
- [x] Load and visualize Survey data tables
- [x] Load and visualize Bipolar electrodes table
- [x] Copy Survey data tables to new file
- [x] Copy Bipolar electrodes table to new file
- [x] ERP with bipolar

fix #47 #48 #50

### CODE REVIEW

**Comment by luiztauffer** [comment]
@bendichter 
this depends on the new release of `ndx-bipolar-scheme` for tests to pass

File: ecogvis/functions/nwb_copy_file.py [hunk 1]
```diff
@@ -381,3 +458,52 @@ def copy_obj(obj_old, nwb_old, nwb_new):
...
+                    vas_relative_pain_intensity_rating=obj_old['vas_relative_pain_intensity_rating'].data[i],
+                    vas_pain_unpleasantness=obj_old['vas_pain_unpleasantness'].data[i],
+                    unix_timestamp=obj_old['unix_timestamp'].data[i],
+                )
+            return vas_survey_table
+
+        elif obj_old.name == 'mpq_survey_table':
+            n_rows = len(obj_old['throbbing'].data[:])
+            for i in range(n_rows):
+                mpq_survey_table.add_row(
```
Comment (bendichter) [inline ecogvis/functions/nwb_copy_file.py:491]: You should be able to do something much more compact for these. Something like:

`mpq_survey_table.add_row({key: val.data[i] for key, val in obj_old.items()})`

File: ecogvis/functions/nwb_copy_file.py [hunk 2]
```diff
@@ -381,3 +458,52 @@ def copy_obj(obj_old, nwb_old, nwb_new):
...
+                    vas_pain_intensity_rating=obj_old['vas_pain_intensity_rating'].data[i],
+                    vas_pain_relief_rating=obj_old['vas_pain_relief_rating'].data[i],
+                    vas_relative_pain_intensity_rating=obj_old['vas_relative_pain_intensity_rating'].data[i],
+                    vas_pain_unpleasantness=obj_old['vas_pain_unpleasantness'].data[i],
+                    unix_timestamp=obj_old['unix_timestamp'].data[i],
+                )
+            return vas_survey_table
+
+        elif obj_old.name == 'mpq_survey_table':
+            n_rows = len(obj_old['throbbing'].data[:])
```
Comment (bendichter) [inline ecogvis/functions/nwb_copy_file.py:489]:
```suggestion
            n_rows = len(obj_old['throbbing'].data)
```

**Comment by bendichter** [comment]
@luiztauffer I've now made you a maintainer of ndx-bipolar-scheme

File: ecogvis/functions/nwb_copy_file.py (hunk 1 above)
Comment (luiztauffer) [reply ecogvis/functions/nwb_copy_file.py]: solved with 9e76d92af879526cdc0b8c64e46b98857f903f2b

File: ecogvis/functions/nwb_copy_file.py (hunk 2 above)
Comment (luiztauffer) [reply ecogvis/functions/nwb_copy_file.py]: solved with 9e76d92af879526cdc0b8c64e46b98857f903f2b

**Pull request merged**